src/test/test-copy.c

```diff
@@ -52,8 +52,8 @@ static void test_copy_file_fd(void) {
         char in_fn[] = "/tmp/test-copy-file-fd-XXXXXX";
         char out_fn[] = "/tmp/test-copy-file-fd-XXXXXX";
         _cleanup_close_ int in_fd = -1, out_fd = -1;
-        char text[] = "boohoo\nfoo\n\tbar\n";
-        char buf[64] = {0};
+        const char *text = "boohoo\nfoo\n\tbar\n";
```

Comment:
not that it matters at all, but you increased footprint by 8 bytes here, since you now declare both a string and a ptr, and previously just a string

Reply:
Yeah, but `const char *` is nicer than `#define`.

Reply:
nah, i was suggesting: 

```c
static const char text[] = "boohoo…";
```
that defines only a string, not a ptr, and does not involve a #define either